templates/accounts/login_signup.html

```diff
@@ -90,6 +101,29 @@ <h1>Hello, Friend!</h1>
     </div>
 </div>
 
+<script>
```

Comment:
Write this script in the `accounts/style.js` in the static folder as that's the corresponding js file for this page.